base/types.go

```diff
@@ -813,6 +859,22 @@ type WrappedMCRequest struct {
 	// cache the MCRequest's size for the sake of stats.
 	// Size = Req.HdrSize() + len(Req.Body) + len(Req.ExtMeta)
 	Size int
+
+	// If VariableVB mode is enabled (not-nil), we need to store the original source VB
+	OrigSrcVB *uint16
```

Comment:
Good to recycle this in mcrequest_pool.go i.e:
```
func (pool *MCRequestPool) cleanReq(req *WrappedMCRequest) *WrappedMCRequest {
...
req.OrigSrcVB = nil
```

Don't think the way this is used, it should cause any problems now, but I would strongly recommend to recycle it.

Reply:
Done